pymongo/pool.py

```diff
@@ -1344,7 +1344,7 @@ def remove_stale_sockets(self, reference_generation):
                     self.requests -= 1
                     self.size_cond.notify()
 
-    def connect(self):
+    def connect(self, error_handler=None):
```

Comment:
Suggest error_handler->handler since that's the name used in get_socket already.

Reply:
Done